docs/content/config/security/rspamd.md

```diff
@@ -92,6 +92,22 @@ Rspamd provides a [web interface][rspamd-docs::web-interface], which contains st
 
 ![Rspamd Web Interface](https://rspamd.com/img/webui.png)
 
+To use the web interface you will need to configure a password, otherwise you won't be able to log in.
+
+??? example "Set a custom password"
+
+    Add this line to [your rspamd `custom-commands.conf` config](#with-the-help-of-a-custom-file) which sets the `password` option of the _controller worker_:
+
+    ```
+    set-option-for-controller password "your hashed password here"
+    ```
+
+    The password hash can be generated via the `rspamadm pw` command:
+
+    ```bash
+    docker exec -it dms rspamadm pw
```

Comment:
From searching docs I get that the commands shown in the docs are either

`docker exec -it <CONTAINER_NAME>` or `docker exec -it CONTAINER_NAME`

and `docker exec -it mailserver`

I'd stick with `docker exec -it <CONTAINER_NAME>` since it's the first command example that you come across in the docs as it's in the [`Usage` section](https://docker-mailserver.github.io/docker-mailserver/latest/usage/#get-up-and-running).